docs/store/filecoin-plus.md

```diff
@@ -102,13 +102,13 @@ _Note: [Lotus-shed](https://github.com/filecoin-project/lotus/tree/master/cmd/lo
 lotus-shed verifreg check-client f00000
 ```
 
-### Finding miners to take FIL+ deals
+### Finding storage providers to take Fil+ deals
```

Comment:
This is a tricky one. _FIL_ is shorthand for _filecoin_, which is the cryptocurrency of the _Filecoin_ network. I wouldn't use _FIL_ to represent the _Filecoin Plus_ program. Saying that, if the program is referred to as _Fil+_ in other places, then that's fine. Not ideal, but it's not terrible.

Reply:
Right - this is why we are switching from FIL to Fil everywhere, so at least the capitalization is different / slightly less misleading. Agree that maybe we need to switch to something even more different, but thats a longer term conversation.

Reply:
Sounds good.